### ISSUE TITLE
'soffice' not found

### ISSUE THREAD

**Issue by Jacqland** [gorkang/html2latex#4]
I'm having the same issue as the previous issue (that has since been closed), but installing LibreOffice has not solved it. The full error is:
```
Error in system(paste(soffice, "--convert-to odt", filename), intern = TRUE) : 
  'soffice' not found
```
If I modify the function and use the Mac = TRUE option to force soffice to point to the file path location on my PC (C:/Program Files/LibreOffice/program/soffice.exe), it gives a different error:
```
Error in system(paste0(file.path(path_w2l, w2l_file), " ", odt_file),  : 
  'C:/Users/[Username]/Documents/R/win-library/4.0/html2latex/extdata/w2l' not found
```
though w2l (and writer2latex.jar) are both in the listed directory.

**Comment by gorkang** [comment]
The package has only been tested on Linux and Mac. I have no access to a Windows system to check... and it's been so long since the last time I used Windows that I am not sure where to start.

The two critical commands are:

```
      invisible(system(paste(soffice, "--convert-to odt", filename), intern = TRUE)) # HTML to ODT
      invisible(shell(paste0(file.path(path_w2l, w2l_file), " ", odt_file), intern = TRUE))  # ODT to TEX

```

So, if you manage to launch in a windows shell/terminal the equivalent to:

`soffice --convert-to odt filename.html`

And then: 

`w2l w2l_file odt_file`

Let me know how the final commands look like, and I can include them in the package.

**Comment by Jacqland** [comment]
Sorry, getting R to talk directly to the terminal is beyond my skill level. I gave it a try but I'm just not sure how to do it. :(

Maybe updating the requirements to list that it works on mac and linux-based systems only, for now? Thank you for the response anyways!

**Comment by gorkang** [comment]
No need to use R at all. If you just try on a Windows sell/terminal the commands below with the appropriate paths and manage to make it work, let me know:

- `soffice --convert-to odt filename.html`
- `w2l w2l_file odt_file`

In any case, I added what you suggested in the README. Thanks for reporting.

**Comment by gorkang** [comment]
Dear @Jacqland I made some changes so it should now work on Windows (given you have Libreoffice and Java installed).

I tested it in a Virtual machine and seems to work (except for the tex to pdf conversion, which requires installing pdflatex.exe. See https://tex.stackexchange.com/questions/49569/where-to-download-pdflatex-exe).

If you want to give it a go, install the last version: remotes::install_github("gorkang/html2latex")

Cheers,
